packages/form-js-viewer/src/render/components/form-fields/Table.js

```diff
@@ -56,7 +56,7 @@ export function Table(props) {
   );
   const columnKeys = evaluatedColumns.map(({ key }) => key);
   const evaluatedDataSource = useExpressionEvaluation(dataSource);
-  const data = Array.isArray(evaluatedDataSource) ? evaluatedDataSource : [];
+  const data = Array.isArray(evaluatedDataSource) ? evaluatedDataSource.filter(i => i) : [];
```

Comment:
Do you want to filter out all [falsy values](https://developer.mozilla.org/en-US/docs/Glossary/Falsy)? Like 0, false, empty string, etc?

Reply:
In this case the expected type is object, but you're right it's more logical to filter by undefined (actually our target is <empty>, but undefined will filter for that as well). Made the change.